runtime/runtime-common/Cargo.toml

```diff
@@ -101,6 +101,7 @@ eth-precompile = [
 	"pallet-evm-eth-receipt-provider",
 	"pallet-receipt-registry",
 ]
+brooklyn = []
```

Comment:
do we have some pallet for feature brooklyn

Reply:
Discussed in sSlack